dissect/target/helpers/record.py

```diff
@@ -149,9 +149,8 @@ def DynamicDescriptor(types):  # noqa
     ("string", "type"),
     ("boolean", "enabled"),
     ("net.ipaddress[]", "dns"),
-    ("net.ipaddress[]", "ip"),
+    ("net.ipinterface[]", "interface"),
```

Comment:
I keep stumbling over this field name but I can't quite think of something nicer, perhaps one of:
- `cidr`
- `address`

?